interface/src/app/gene/selected/selected-genes-panel.tpl.html

```diff
@@ -1,26 +1,15 @@
-<div class="row">
-  <div class="col-sm-12">
-    <div class="panel panel-default">
-      <div class="panel-heading">
-        <h3>
-          Selected Genes
-          <span class="pull-right">
-            <button class="btn btn-success"
-              ng-click="sendToNetwork()">
-              View Genes as Network
-            </button>
-          </span>
-        </h3>
-      </div>    
-      <div class="panel-body">
-        <table class="table table-striped"> 
-          <tr ng-repeat="gene in selectedGenes">
-            <td>
-              <selected-gene-button gene="gene"></selected-gene-button>
-            </td>
-          </tr>     
-        </table>
-      </div>
-    </div>
-  </div>
+<h3>Selected Genes</h3>
+<br>
+
+<div>
+  <span ng-repeat="gene in selectedGenes">
+    <selected-gene-button gene="gene"></selected-gene-button>
+  </span>
 </div>
+<br>
+
+<span class="pull-right">
+  <button class="btn btn-success btn-lg" ng-click="sendToNetwork()">
+    View Genes as Network <i class="fa fa-check"></i>
+  </button>
+</span>
```

Comment:
This is a nice. Much tighter and more component-like now. I think I'd still prefer to see things like headers (lines 1-3) deferred to the view that assembles the components. That keeps overall layout-oriented concerns where they can be best addressed while making the component as compact and re-usable as possible. Maybe to be addressed as a separate refactoring issue, as discussed above.